environ.py

```diff
@@ -1,25 +1,28 @@
 # standard libraries
```

Comment:
The more I use it, the more I think the `environ` module should be renamed `configuration`.  Because it doesn't represent the actual environment.  It's a combination of the environment and configuration settings.

Reply:
I'm opening to discussing this, though I am not a fan of `configuration`, particularly. Please open a separate issue.

Reply:
OK.  See #94.